docs/en/enterprise-edition/policy-reference/ibm-policies/ibm-iam-policies/bc-ibm-2-5.adoc

```diff
@@ -0,0 +1,46 @@
+== IBM Cloud Service ID creation is not restricted in account settings in Terraform
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| TBD
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/graph_checks/ibm/IBM_RestrictServiceIDCreationInAccountSettings.yaml[CKV2_IBM_5]
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|Terraform,TerraformPlan
+
+|===
+
+=== Description
+
+This policy checks to ensure that the creation of Service IDs is restricted within account settings. Service IDs are used to identify and authenticate services or applications in order to allow them to interact with other services or resources. Ensuring that their creation is restricted helps to prevent unauthorized access or mismanagement of service identities, which could lead to security vulnerabilities. By limiting who can create Service IDs, you are enhancing the overall security posture of the account and mitigating the risk of potential breaches or misconfigurations.
```

Comment:
This policy detects whether Service ID creation is unrestricted in account settings. Service IDs are used to identify and authenticate services or applications, allowing them to interact with other services or resources. Unrestricted creation can increase the risk of unauthorized access and mismanagement of service identities. By restricting creation, you can enhance account security and mitigate potential breaches or misconfigurations.